docs/nonlocal-networks.rst

```diff
@@ -0,0 +1,138 @@
+
+The connection settings for non-local networks must be defined in ``brownie-config.json``.
+
+First, for each network you want to configure, create a new section in the network.networks section as below:
+
+.. code-block:: javascript
+
+    "networks": {
+        .
+        .
+        "ropsten": {
+            "host": "http://ropsten.infura.io/v3/[yourInfuraApiKey]"
+        }
+        "rinkeby":...
+        .
+        .
+    }
+
+If you want to change the default network that brownie connects to, you need to update the network.default field as below:
+
+.. code-block:: javascript
+
+    "network": {
+        "default": "ropsten",
+        .
+        .
+    }
+
+Launching and Connecting to Networks
+====================================
+
+Using the CLI
+-------------
+
+Brownie will connect to whichever network is set as "default" in ``brownie-config.json``.  
+
+To connect to any other network that has been defined in ``brownie-config.json``, you must specify the `--network` flag when launching Brownie as below:
+::
+    $ brownie --network ropsten
+
+
+Each time Brownie is loaded, it will first attempt to connect to the ``host`` address to determine if the RPC client is already active.
+
+Using the ``brownie.netowrk`` methods
+-------------------------------------
+
+You can interact with any network defined in ``brownie-config.json`` programatically using the ``brownie.network.main`` methods.
+
+Connecting to a network:
+
+.. code-block:: python
+
+    >>> network.connect('ropsten')    
+    >>> network.is_connected()
+    True
+    >>> network.show_active()
+    'ropsten'
+
+Disconnecting from a network:
+
+.. code-block:: python
+
+    >>> network.disconnect()
+    >>> network.is_connected()
+    False
+
+Interacting with non-local networks
+==============================
+
+``brownie.module.rpc``
+--------
+
+The :ref:`rpc` module is unavailable when working with non-local networks.
+
+Accounts
+--------
+
+When loading an account for interacting with a non-local network, you must provide the private key when loading the account in order to be able to sign transactions or deploy contracts
+
+.. code-block:: python
+
+        >>> accounts.add('8fa2fdfb89003176a16b707fc860d0881da0d1d8248af210df12d37860996fb2')
+        <Account object '0xc1826925377b4103cC92DeeCDF6F96A03142F37a'>
+        >>> accounts[0].balance()
+        17722750299000000000
+
+Once an account is added to the account object, the ``accounts.save`` and ``accounts.load`` can be used to save the accounts to an encrypted keystore and then load for later use.
+
+Transactions
+------------
+
+* Transaction status
+When submitting a transaction on non-local networks, blocks are not immediately so transactions will likewise not be immediately confirmed. 
+A :ref:`api-network-tx` object is provided immediately and can be stored to unique variables though ``TransactionReceipt.status`` will be ``-1`` until the transaction is mined and either succeeds or reverts.  
```

Comment:
A `TransactionReceipt` is not returned immediately by default, Brownie will display a "waiting for confirmation..." message until it confirms.  In the console, a user can press `CTRL-C` while waiting, and then they are immediately returned the pending `TransactionReceipt`.

Related: https://eth-brownie.readthedocs.io/en/latest/interaction.html#unconfirmed-transactions

Reply:
Clarifications added.